configuration/sitemaps.md

```diff
@@ -447,6 +448,24 @@ List item=<itemname> [label="<labelname>"] [icon="<iconname>"] [separator=""]
 Splits a String Item at each separator into multiple rows.
 -->
 
+### Element Type 'MapView'
```

Comment:
[Mapview](https://github.com/eclipse/smarthome/pull/277/files#diff-10aa8fe84e9119cfe8cac6ea022ff706)?

I don't think Mapview should be at the end of the list. Priority-wise I'd say you can move it somewhere around Webview